tests/conftest.py

```diff
@@ -1,7 +1,11 @@
 import httpx
 import pytest
 from _pytest.logging import LogCaptureFixture
-from loguru import logger
+
+try:
+    from loguru import logger
+except ModuleNotFoundError:
+    logger = None
```

Comment:
For me, it would also be fine to include loguru as test dependency in the `environment_dev.yml` . Up to you!

Reply:
Yes. Better including as dep. Done